### PR TITLE
adding record_id to each split instrument

### PR DESCRIPTION
**Pull request by wdefreitas**
Adding record_id and event name to each split instrument

### CODE REVIEW

**Comment by wdefreitas** [comment]
Also added redcap_event_name to each instrument

File: DESCRIPTION [hunk 1]
```diff
@@ -38,23 +45,26 @@ Description:
...
 License: MIT + file LICENSE
 Encoding: UTF-8
-RoxygenNote: 7.1.2
+RoxygenNote: 7.2.1
 Depends: R (>= 3.5.0)
 Imports: 
+    dplyr,
+    janitor,
+    labelVector,
+    magrittr,
```
Comment (gabrielodom) [inline DESCRIPTION:54]: Why are we using two pipes?

File: R/import_instruments_redcapr.R [hunk 2]
```diff
@@ -0,0 +1,164 @@
+
+#' @title relabel
```
Comment (gabrielodom) [inline R/import_instruments_redcapr.R:2]: Helper functions go below the main function in the file

File: R/import_instruments.R [hunk 3]
```diff
@@ -0,0 +1,163 @@
+
```
Comment (gabrielodom) [inline R/import_instruments.R:1]: Lintr??

File: R/make_choose_all_table.R [hunk 4]
```diff
@@ -19,29 +19,34 @@ getLabel2 <- function(data, aVariable) {
...
 #'
-## @examples make_choose_all_table(redcap, "ingredients")
+## @examples
+##\dontrun{
+##  make_choose_all_table(redcap, "ingredients")
+##  }
 make_choose_all_table <- function(df, variable) {
-  # . <- NULL # kludge to get CMD Check to pass with nonstandard evaulation
+  
+  # . <- NULL # kludge to get CMD Check to pass with nonstandard evaluation
```
Comment (gabrielodom) [inline R/make_choose_all_table.R:44]: Bless your heart

**Review by gabrielodom — COMMENTED** [review]
Overall, most problems will be fixed in version 1.x enhancements, or perhaps in version 2.0. To me, I have no problem approving this for release.

**Pull request merged**